### ISSUE TITLE
Panic when running Nakama 2.7.0 on Windows

### ISSUE THREAD

**Issue by joshgarnett** [heroiclabs/nakama#388]
When running Nakama 2.7.0 with a Go Plugin, we encounter the following crash on windows.

```
nakama | {"level":"info","ts":"2019-10-22T20:54:01.327Z","msg":"Starting Cluster RPC server","port":7353}
nakama | runtime: unexpected return pc for os/signal.loop called from 0x0
nakama | stack: frame={sp:0xc00005c7c0, fp:0xc00005c7e0} stack=[0xc00005c000,0xc00005c800)
nakama | 000000c00005c6c0:  0000000000000000  0000000000000000
nakama | 000000c00005c6d0:  0000000000000000  0000000000a97046 <runtime.futexsleep+70>
nakama | 000000c00005c6e0:  0000000002cce5a0  0000000000000080
nakama | 000000c00005c6f0:  0000000000000000  0000000000000000
nakama | 000000c00005c700:  0000000000000000  000000c00005c768
nakama | 000000c00005c710:  000000c0000a0480  0000000000000000
nakama | 000000c00005c720:  000000c00005c758  0000000000a76286 <runtime.notetsleep_internal+134>
nakama | 000000c00005c730:  0000000002cce5a0  0000000000000000
nakama | 000000c00005c740:  0000000000000000  0000000000000000
nakama | 000000c00005c750:  0000000000000000  0000000000000000
nakama | 000000c00005c760:  0000000000000000  0000000000000000
nakama | 000000c00005c770:  0000000000000000  0000000000000000
nakama | 000000c00005c780:  0000000000000000  0000000000000000
nakama | 000000c00005c790:  0000000000ab05cc <os/signal.signal_recv+156>  0000000002cce5a0
nakama | 000000c00005c7a0:  ffffffffffffffff  0000000000000000
nakama | 000000c00005c7b0:  000000c00005c7d0  0000000000da02e2 <os/signal.loop+34>
nakama | 000000c00005c7c0: <0000000000000000  0000000000000000
nakama | 000000c00005c7d0:  0000000000000000 !0000000000000000
nakama | 000000c00005c7e0: >0000000000000000  0000000000000000
nakama | 000000c00005c7f0:  0000000000000000  0000000000000000
nakama | fatal error: unknown caller pc
```

This does not crash on OSX or Linux.

**Comment by joshgarnett** [comment]
With @novabyte we tested Go 1.13.3, Alpine 3.10.3, and Debian buster.  The only thing that worked was Debian buster.  My best guess is this is a regression in Go 1.13 and how it interacts with musl libc in Alpine distros.

**Comment by joshgarnett** [comment]
Put together a version of Go 1.13.0 and Nakama that uses Alpine 3.9.4 at https://github.com/heroiclabs/nakama/compare/master...joshgarnett:v2.7.0-alpine3.9?expand=1 which does not crash.

**Comment by joshgarnett** [comment]
I've put together a very simple project that triggers this issue when loading a basic plugin at https://github.com/joshgarnett/nakama-2.7.0-go-1.13-crash

**Comment by zyro** [comment]
This will be solved by a36eddb1c09309a0f6a92bfd52599fda3c0b85dd moving to debian-slim base images, and will be part of the next release.
